PyViCare/PyViCareService.py

```diff
@@ -61,4 +64,6 @@ def setProperty(self, property_name: str, action: str, data: Any) -> Any:
 
     def fetch_all_features(self) -> Any:
         url = f'/features/installations/{self.accessor.id}/gateways/{self.accessor.serial}/devices/{self.accessor.device_id}/features/'
+        if self._isGateway():
+            url = f'/features/installations/{self.accessor.id}/gateways/{self.accessor.serial}/features/'
```

Comment:
Can you please add a test that this url gets called for your Gateway device?

Reply:
Is there any sample I can reuse?

Reply:
Any suggestion how to test this?

Reply:
Just copy this function and set the roles on the service like you did on your other test

https://github.com/somm15/PyViCare/blob/16eb10d1c5fdb372a669705b54e34bb222b2057b/tests/test_PyViCareService.py#L14